sros2/sros2/verb/create_permission.py

```diff
@@ -31,17 +31,17 @@ class CreatePermissionVerb(VerbExtension):
     """Create permission."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg = parser.add_argument('-k', '--keystore-root-path', help='root path of keystore')
         arg.completer = DirectoriesCompleter()
-        parser.add_argument('NAME', help='key name, aka ROS node name')
-        arg = parser.add_argument(
-            'POLICY_FILE_PATH', help='path of the policy xml file')
-        arg.completer = FilesCompleter(
-            allowednames=('xml'), directories=False)
+        parser.add_argument(
+            '-c', '--security-context', help='identity, aka ROS security contexts path')
+        arg = parser.add_argument('-p', '--policy-files', help='list of policy xml file paths')
```

Comment:
This is still misleading IMO as the underlying function takes a single policy file.. Can you please use the original help text and a singular form for the long flag name ?

Reply:
Sorry, see https://github.com/ros2/sros2/pull/177/commits/c23bd7b43db0b51bbe387b71124200a60a570fda.